glang.spec

```diff
@@ -0,0 +1,61 @@
+Name:           glang
+Version:        0.1.3
+Release:        %autorelease
+Summary:        A Graphene front-end for LLVM
+
+License:        AGPL-3.0-only
+URL:            https://github.com/DaveDuck321/GrapheneLang
+Source0:        %{url}/archive/refs/heads/dev_hatch.tar.gz
```

Comment:
How are we going to keep this updated? Do we just tag releases and update the URL in this file?

Also is this python ecosystem or Fedora-specific?

Reply:
> How are we going to keep this updated? Do we just tag releases and update the URL in this file?

Yeah we can tag releases and have the URL update automatically with the `%{version}` macro. Eventually I'd like to move the version to a single place and have the packaging files just read it from there.

We can also build on every commit if we want, instead of releases.

Another option was to have the spec file in its own repo (which is what distro package maintainers normally do), but then you have to manually update it on every release. 

> Also is this python ecosystem or Fedora-specific?

The spec file is just for RPMs. So Fedora and RHEL (+clones), as well as SUSE.